.travis.yml

```diff
@@ -17,8 +17,8 @@ branches:
 env:
   global:
     # Release version has to start with real version (7.1.0-....) for the docker image to build successfully.
-    - RELEASE_VERSION=7.2.0-A4
-    - DEVELOPMENT_VERSION=7.2.0-A5-SNAPSHOT
+    - RELEASE_VERSION=7.2.0
+    - DEVELOPMENT_VERSION=7.2.0-A6-SNAPSHOT
```

Comment:
After we release 7.2.0 then the next release version will be 7.2.0.1, 7.2.1 or 7.3.0 (depending on the branch). I don't think we want this to be set to 7.2.0-A6-SNAPSHOT.

Reply:
Thanks Tom, I had the same suspicion but couldn't find what the right version would be in this case. @montgolfiere what do you think Jan?

Reply:
We need all three, because ultimately we need three branches after 7.2.0.